### DISCUSSION TITLE
Episode 129: Circle is Charizard! 🔥

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#20] [Episode]
Episode Link: https://adspthepodcast.com/2023/05/12/Episode-129.html

Feel free to write comments or ask questions 🙂

**Comment by waydan** [comment]
Charmander struggled with the first two gyms (rock and water). Bulbasaur was nice early, but kinda meh later. I always my favorite, though.

Super important feedback...

**Comment by vmiheer** [comment]
Thank you so much for linking https://github.com/seanbaxter/circle/issues/105.
